pkg/client/client.go

```diff
@@ -107,14 +107,17 @@ func NewDefaultClientWithTimeout(timeout time.Duration) (*Client, error) {
 // If host is nil then use SockPath provided by CILIUM_SOCK
 // or the cilium default SockPath
 func NewClient(host string) (*Client, error) {
-	clientTrans, err := NewRuntime(host)
+	clientTrans, err := NewRuntime(host, "")
 	return &Client{*clientapi.New(clientTrans, strfmt.Default)}, err
 }
 
-func NewRuntime(host string) (*runtime_client.Runtime, error) {
+func NewRuntime(host, basePath string) (*runtime_client.Runtime, error) {
```

Comment:
we might want to start using the options pattern here

Reply:
Agreed, see https://github.com/cilium/cilium/pull/29271